README.md

```diff
@@ -69,7 +69,7 @@ ow.actions.invoke('sample').then(result => console.log(result))
 ```
 
 #### using 3rd party authentication handler
-You can specify an authentication handler in `options.auth_handler` this is an object that provides a function `getAuthHeader` that returns a Promise or String to be used in the `Authorization` http header for every http request.
+You can specify an authentication handler in `options.auth_handler` this is an object that provides a function `getAuthHeader` that returns a Promise resolving to a string to be used in the `Authorization` http header for every http request.
```

Comment:
The code, as written today, will throw if `options.auth_handler` returns a `string`

**Invocation of `auth_handler`**:
https://github.com/apache/openwhisk-client-js/blob/532236924ef97c00542bca78165312c105525da6/lib/client.js#L209-L216

**Usage of result**:
https://github.com/apache/openwhisk-client-js/blob/532236924ef97c00542bca78165312c105525da6/lib/client.js#L159